conditions/src/main/scala/explore/conditions/Conditions.scala

```diff
@@ -7,17 +7,32 @@ import japgolly.scalajs.react._
 import japgolly.scalajs.react.vdom.html_<^._
 import react.semanticui.elements.button._
 import react.semanticui.colors._
+// import crystal.react.io.implicits._
+import explore.model._
+import scala.scalajs.js
+import js.UndefOr._
+import cats.effect._
 
 object Conditions {
 
+  @inline implicit def io2UndefOrCallback[A](io: IO[A]): js.UndefOr[Callback] = Callback {
+    io.unsafeRunAsyncAndForget()
+  }
+
   private val component =
     ScalaComponent
       .builder[Unit]("Conditions")
       .render { _ =>
         <.div(
-          Button("IQ"),
-          Button(color = Blue)("Button", "Btn"),
-          Button("Button", "Dec")
+          <.div(
+            Button("IQ"),
+            Button(color = Blue)("Button", "Btn"),
+            Button("Button", "Dec")
+          ),
+          <.div(
+            Button(onClick = Actions.TargetActionsIO.set(Target.M81)).withChildren(Seq("M81")),
```

Comment:
You should be able to do
`Button(...)("M81")`
if you import `react.common._` or `react.common.syntax._`

Reply:
Great! I will do that, thank you for the tip.